.github/workflows/commit-checker.yaml

```diff
@@ -0,0 +1,46 @@
+name: Unit Tests
+on:
+  pull_request:
```

Comment:
As mentioned in the previous comment. try updating this event trigger to `pull_request_target`.